### ISSUE TITLE
Error while filling the input in the Preline UI component

### ISSUE THREAD

**Issue by ardaalkan** [htmlstreamofficial/preline#370]
Currently I'm using PrelineUI("preline": "^2.0.1",) and tailwindcss("tailwindcss": "^3.3.5",). I have more than one input in an preline accordion component. Even though I haven't made any checks, inputs sometimes do not allow English characters to be written when trying to fill them. I don't know what's causing the problem.

If inputs are autofilled, an error message appears. When you try to fill it manually, for example with "A", a warning appears in the console with the phrase 'KeyCode: KeyA'.

As a result, I will be very happy if I fill in the inputs successfully and do not receive the warning and this error that I receive from the library.

More detailed
https://stackoverflow.com/questions/78495265/error-while-filling-the-input-in-the-preline-ui-component

**Comment by jahaganiev** [comment]
Hey @ardaalkan - we couldn't detect any issues, please try out with the latest version of Preline UI. Thanks!

**Comment by ardaalkan** [comment]
Hey @jahaganiev, You couldn't detect any issues but i detect.

I use many preline components in one project. While using many dropdown components within a page, dropdown and dropdown 'Auto close behavior' worked together, but if the auto close feature was triggered even once, it would block the auto close feature, I don't know why!. 

When I removed all the auto-close dropdowns I used in the project, the problem was magically resolved in a way that I still don't understand yet.

and friends, instead of dropdowns with auto-close feature, use a dropdown that does not have this feature or make a new one yourself. Don't forget to write CSS animations :)

thanks for your interest.
